client/src/pages/StartPage.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import styled from '@emotion/styled';
 import { Link } from 'react-router-dom';
```

Comment:
Deepscan ist still failing because this import is unused.

Reply:
Oh thanks, Oleg, I already remove it.  :)